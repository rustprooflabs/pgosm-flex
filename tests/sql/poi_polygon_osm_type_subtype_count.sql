SELECT osm_type COLLATE "C", osm_subtype COLLATE "C", COUNT(*)
    FROM osm.poi_polygon
    GROUP BY osm_type COLLATE "C", osm_subtype COLLATE "C"
    ORDER BY osm_type COLLATE "C", osm_subtype COLLATE "C"
;