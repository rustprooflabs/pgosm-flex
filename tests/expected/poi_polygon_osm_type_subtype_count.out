amenity|Gym|1
amenity|Personal Training|1
amenity|alcohol|1
amenity|arts_centre|4
amenity|bank|19
amenity|bar|7
amenity|bench|9
amenity|bicycle_parking|2
amenity|bicycle_rental|1
amenity|bicycle_repair_station|1
amenity|biergarten|1
amenity|boat_rental|1
amenity|cafe|10
amenity|car_rental|1
amenity|car_wash|1
amenity|cinema|6
amenity|clinic|2
amenity|college|6
amenity|community_centre|10
amenity|conference_centre|1
amenity|courthouse|4
amenity|coworking_space|1
amenity|dentist|2
amenity|doctors|4
amenity|embassy|44
amenity|fast_food|42
amenity|fire_station|20
amenity|fountain|9
amenity|fuel|35
amenity|funeral_home|1
amenity|hospital|12
amenity|kindergarten|12
amenity|library|30
amenity|marketplace|4
amenity|nightclub|3
amenity|nursing_home|1
amenity|office|1
amenity|parking|6555
amenity|parking_entrance|1
amenity|parking_space|38
amenity|pharmacy|7
amenity|place_of_worship|233
amenity|planetarium|1
amenity|police|13
amenity|post_office|8
amenity|pub|4
amenity|restaurant|74
amenity|school|148
amenity|shelter|4
amenity|social_facility|4
amenity|stripclub|1
amenity|theatre|15
amenity|toilets|11
amenity|townhall|1
amenity|university|43
amenity|veterinary|2
amenity|waste_disposal|1
building|39|1
building|44|1
building|Gym|1
building|Hotel|1
building|Utility_Suubstation|1
building|amenity=fire_station;building=government|1
building|apartments|172
building|burial_vault|21
building|church|4
building|civic|2
building|college|3
building|commercial|34
building|condominium|3
building|construction|2
building|dormitory|28
building|garage|2
building|government|4
building|greenhouse|1
building|hospital|2
building|hotel|6
building|house|15
building|industrial|3
building|manufacture|1
building|no|2
building|office|45
building|public|6
building|residential|56
building|retail|3
building|roof|1
building|school|30
building|service|2
building|stable|1
building|temple|1
building|terrace|3
building|train_station|3
building|university|147
building|warehouse|1
building|yes|942
historic|archaeological_site|2
historic|house|1
historic|memorial|7
historic|monument|9
landuse|cemetery|24
landuse|railway|2
landuse|village_green|3
leisure|bleachers|2
leisure|dog_park|10
leisure|events|1
leisure|fitness_centre|1
leisure|garden|182
leisure|golf_course|4
leisure|ice_rink|1
leisure|marina|10
leisure|miniature_golf|2
leisure|nature_reserve|2
leisure|outdoor_seating|5
leisure|park|864
leisure|picnic_table|3
leisure|pitch|448
leisure|playground|122
leisure|sports_centre|8
leisure|stadium|10
leisure|swimming_pool|48
leisure|track|26
leisure|water_park|2
leisure|yes|1
man_made|chimney|6
man_made|pier|28
shop|alcohol|15
shop|art|1
shop|art;paper|1
shop|bakery|3
shop|beauty|7
shop|bicycle|1
shop|books|1
shop|car|1
shop|car_parts|3
shop|car_repair|2
shop|chemist|2
shop|clothes|8
shop|convenience|35
shop|copyshop|1
shop|cosmetics|2
shop|deli|1
shop|department_store|5
shop|doityourself|2
shop|dry_cleaning|3
shop|electronics|1
shop|florist|1
shop|frame|2
shop|funeral_directors|4
shop|garden_centre|1
shop|gift|5
shop|greengrocer|1
shop|hairdresser|10
shop|interior_decoration|2
shop|laundry|4
shop|mall|2
shop|mobile_phone|1
shop|paint|1
shop|pet|1
shop|religion|1
shop|seafood|4
shop|shoes|2
shop|sports|2
shop|supermarket|15
shop|ticket|1
shop|tobacco|2
shop|tyres|1
shop|variety_store|3
shop|wholesale|1
shop|yes|1
tourism|artwork|18
tourism|attraction|41
tourism|hotel|2
tourism|information|2
tourism|picnic_site|6
tourism|yes|1
tourism|zoo|3
